data/common.yaml

```diff
@@ -1,4 +1,29 @@
 ---
+profile_backup::client::enabled: true
+profile_backup::client::encryption_passphrase: "CHANGE ME"
+profile_backup::client::job_cron_schedule:
+  hour: 0
+  minute: 30
+profile_backup::client::prune_settings:
+  #"keep-secondly": 1
```

Comment:
lint: keep commented code out of production

also applies to other lines in this file